app/routes/transactions.js

```diff
@@ -1,4 +1,5 @@
 import Route from '@ember/routing/route';
 
 export default Route.extend({
-});
+
```

Comment:
again brief comment that this route is under development or delete if not needed.